### ISSUE TITLE
Elevated urine suberic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6024]
**Preferred term label:** Elevated urine suberic acid concentration

**Synonyms** Elevated urine octanedioic acid concentration

**Definition (free text, please give PubMed ID)** Concentration of suberic acid in the urine is above the normal range., PMID: 6466694; PMID: 5031780

**Parent term (use hpo.jax.org/app)** Dicarboxylic aciduria HP:0003215

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:9300

**Comment by pnrobinson** [comment]
added term
